source/fundamentals/connection/tuning.txt

```diff
@@ -0,0 +1,166 @@
+=================
+Connection Storms
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _connection-tuning-java:
+
+Overview
+--------
+
+In this guide, you can learn about **connection storms** and settings that
+can help avoid them.
+
+Connection Storms
+-----------------
+
+A **connection storm** is an event in which application servers
+start an increasing number of connection requests, but the host server
+CPU is unable to schedule them.
```

Comment:
I think it would be valuable to add it more naturally than the example, but I can't think of something concrete to add since there are many scenarios that can lead to a connection storm.